internal/cli/atlas/serverless/backup/restores/describe.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package restores
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	id          string
+	clusterName string
+	store       store.ServerlessRestoreJobsDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var restoreDescribeTemplate = `ID	SNAPSHOT	CLUSTER	TYPE	EXPIRES AT	URLs
+{{.ID}}	{{.SnapshotID}}	{{.TargetClusterName}}	{{.DeliveryType}}	{{.ExpiresAt}}	{{range $index, $element := .DeliveryURL}}{{if $index}}; {{end}}{{$element}}{{end}}
+`
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.ServerlessRestoreJob(opts.ConfigProjectID(), opts.clusterName, opts.id)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas backup(s) restore(s) job(s) describe <ID>.
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describe a cloud backup restore job.",
+		Long:  fmt.Sprintf(usage.RequiredRole, "Project Owner"),
+		Args:  require.NoArgs,
+		Example: fmt.Sprintf(`  # Return the details for the continuous backup restore job with the ID 507f1f77bcf86cd799439011 for the serverless isntance named Cluster0:
+  %s serverless backup restore describe --restoreJobId 507f1f77bcf86cd799439011 --clusterName Cluster0`, cli.ExampleAtlasEntryPoint()),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), restoreDescribeTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.id = args[0]
```

Comment:
is the id required? if not, we should check that this will not fail if the arg is not pass. if yes, we should use `require.ExactArgs(1)` and update the Example to pass the ID as arg

Reply:
that was a copy/paste mistake, I have no args

Reply:
I see that we removed the arg from the command. 
We have this rule in the CLI
> Commands follow a  [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer)  approach to match the APIs, whenever possible. For that reason, command arguments tend to match the path and query params of the APIs, with the last param being a required argument and the rest handled via flag options.


This rule is also in our [WIKi](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=100255555#MongoDBAtlasCLI(AtlasCLI)andMongoDBCLI(mongocli)-Addinganewcommand)


I believe the last param of the request `restoreJobId` should be a required arg for the command.

Reply:
It was agreed in our spec https://docs.google.com/document/d/14mSR3XepNwBKzREkrpfLYbANMIi2Glc54n_uawylico/edit

Reply:
Ops I am sorry, I missed it 🤦  I will add a follow-up discussion on slack just to make sure we are all on the same page. No need to address this here